src/server/vfs/win/vfs_win.cpp

```diff
@@ -654,11 +654,11 @@ bool VfsWin::status(const QString &filePath, bool &isPlaceholder, bool &isHydrat
 
 bool VfsWin::fileStatusChanged(const QString &path, SyncFileStatus status) {
     LOGW_DEBUG(logger(), L"fileStatusChanged: " << Utility::formatSyncPath(QStr2Path(path)).c_str() << L" status = "
-                                                << Utility::s2ws(Utility::syncFileStatus2Str(status)).c_str());
+                                                << Utility::s2ws(Utility::SyncFileInstruction2Str(status)).c_str());
```

Comment:
Why using the upper case `S` in the function name?

Reply:
My bad, I made the correction in commit: [24387cd](https://github.com/Infomaniak/desktop-kDrive/pull/174/commits/24387cd7af976038b398286c11a65d90a5aeba01)